decidim-core/lib/decidim/map/provider/autocomplete/osm.rb

```diff
@@ -10,7 +10,7 @@ class Osm < ::Decidim::Map::Autocomplete
           class Builder < Decidim::Map::Autocomplete::Builder
             # @see Decidim::Map::FrontendUtility::Builder#javascript_snippets
             def javascript_snippets
-              template.javascript_pack_tag("decidim_geocoding_provider_photon", defer: false)
+              template.append_javascript_pack_tag("decidim_geocoding_provider_photon")
```

Comment:
If we let the `defer: false` as a call for method, it will generate errors like: 

```
Uncaught ReferenceError: $ is not defined
    js here.js:9
    Webpack 7
```
Even if we move around and remove that $ call, we will end up calling some leaflet function that uses Jquery resulting in an error. 


Removing the defer, it would load normally, any js errors.

Reply:
I am marking this comment as resolved, as the question has been answered.